live-site/components/event-schedule-section/EventScheduleSection.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { mockData } from './EventScheduleMockData';
+
+import {
+  StyledBoldSubText,
+  EventScheduleSectionContainer,
+  StyledEventScheduleHeader,
+  StyledEventScheduleSubText,
+  StyledAddCalenderButton,
+  ButtonContainer,
+  EventScheduleTextButtonContainer
+} from './EventScheduleSection.styles';
+import EventScheduleTab from './EventScheduleTab';
+
+const EventScheduleSection: React.FC = () => {
+  return (
+    <EventScheduleSectionContainer>
+      <StyledEventScheduleHeader>Event Schedule</StyledEventScheduleHeader>
+      <EventScheduleTextButtonContainer>
+        <StyledEventScheduleSubText>
+          Dates and times are displayed in your local timezone. Schedule in EST
+          can be found here.
+          <br />
+          The password for all Zoom meetings can be found pinned in the
+          #announcements Slack channel.
+          <br />
+          <StyledBoldSubText>
+            Please be sure to attend all events labeled "Everyone".
+          </StyledBoldSubText>
+        </StyledEventScheduleSubText>
+        <ButtonContainer>
+          <StyledAddCalenderButton
+            btnText="Add to Calender +"
+            btnLink=""
```

Comment:
Not a blocker for getting this ticket merged in but would you mind asking around to find out who is supposed to create this link?

Reply:
and maybe bump them on it if it's not on their radar yet

Reply:
@jzhzhang66 Okay! I asked David and Sreya!